PIM/API/01_Overview.md

```diff
@@ -0,0 +1,227 @@
+
+#### Sample: Success, product created
+
+        {
+            "product": {
+            "id": 1171,
+            "sku": "new-product",
+            "attributeSetId": 102,
+            ...
+            },
+            "success": true,
+            "displayMessage": "The product was successfully created",
+            "displayMessageTitle": "Create product",
+            "error": null,
+            "job_id": null
+        }
+
+> [Info] For a complete list of attributes contained in the product object, see [The product object](./05_ManageProducts.md#the-product-object).
+
+#### Sample: Error, required field missing or empty
+
+        {
+            "success": false,
+            "errno": 10023,
+            "error": "Product could not be created: Attribute set id is not set or empty",
+            "trace": "<pre> (Actindo\\Modules\\Actindo\\PIM\\Exception\\CreateProductFailedException) in ... \/pre>"
+        }
+
+> [Info] For a response schema, refer to the API documentation in your *Actindo Core1 Platform* instance under *Dev Tools > API > Module name > API endpoint*.
+
+[comment]: <> (Add a "general" response schema? Depends on response, ok or error, and resource...)
+
+#### Response codes
+
+Generally, the *Actindo Core1 OpenAPI* uses the standard HTTP status codes to indicate the success of failure of the API requests. The most usual response codes returned are as follows: 
+
+| Code | Description | Notes |
+| ---- | ----------- | ------ |
+| 200  | OK          |  Everything worked as expected      |
+| 400  | Bad request | Unacceptable request, often due to missing required parameter or  header, e.g. Content-Type: application/json, or malformed json |
+| 401  | Unauthorized | Authentication has failed, e.g. due to invalid or missing authentication credentials |
+| 429  | Too many requests   | You sent too many requests in a given amount of time |
+| 530  | Site frozen | The tenant is currently frozen due to maintenance work, module installations or updates |
+| default | Generic error | - | 
+
+For detailed information, see [HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes "[https://en.wikipedia.org/wiki/List_of_HTTP_status_codes]").
+
+
+
+### Authentication
+
+The *Actindo Core1 Platform* supports two workflows of the OAuth 2.0 protocol to retrieve an access token and make authenticated requests. For detailed information on the authentication flow, see [Set up the OAuth authorization](02_GetStarted.md#set-up-the-oauth-authorization).
+
+
+### Tools
+
+You can use different software tools to submit your requests, either your own tool or a third-party tool, such as Postman. For a detailed description on how to set up Postman, see [Set up Postman for the Actindo Core1 OpenAPI](#set-up-postman-for-the-actindo-core1-openapi). 
+
+Regardless of how you submit your requests, you always need to provide the following information:
```

Comment:
Was hat das mit Tools zu tun?Oder meinst Du: Regardless of the tool you use to submit....?